doc/index.rst

```diff
@@ -151,20 +145,20 @@ It also generates a PDF file when :code:`pdf` option is given, which might be us
 .. _width:
 .. _height:
 
-``:width:`` and ``:height:``
-============================
+``:viewport-width:`` and ``:viewport-height:``
+==============================================
 
 You can specify the screen size for a particular screenshot with :code:`width` and :code:`height` parameters.
```

Comment:
Maybe they should be renamed to `viewport-width` and `viewport-height` as well?

Reply:
Nice catch. Done.